best-practices/readonly-nodes.md

```diff
@@ -0,0 +1,131 @@
+    ```
+
+    If you have not configured Placement Rules before, the output file is as follows:
+
+    ```json
+    [
+      {
+        "group_id": "pd",
+        "group_index": 0,
+        "group_override": false,
+        "rules": [
+          {
+            "group_id": "pd",
+            "id": "default",
+            "start_key": "",
+            "end_key": "",
+            "role": "voter",
+            "count": 3
+          }
+        ]
+      }
+    ]
+    ```
+
+2. Store all data on the read-only nodes as a learner. The following example is based on the default configuration.
+
+    ```json
+    [
+      {
+        "group_id": "pd",
+        "group_index": 0,
+        "group_override": false,
+        "rules": [
+          {
+            "group_id": "pd",
+            "id": "default",
+            "start_key": "",
+            "end_key": "",
+            "role": "voter",
+            "count": 3
+          },
+          {
+            "group_id": "pd",
+            "id": "readonly",
+            "start_key": "",
+            "end_key": "",
+            "role": "learner",
+            "count": 1,
+            "label_constraints": [
+              {
+                "key": "$mode",
+                "op": "in",
+                "values": [
+                  "readonly"
+                ]
+              }
+            ],
+            "version": 1
+          }
+        ]
+      }
+    ]
+    ```
+
+3. Use the `pd-ctl config placement-rules` command to write the configuration to PD:
+
+    ```shell
+    pd-ctl config placement-rules rule-bundle save --in="rules.json"
+    ```
+
+> **Note:**
+>
+> - If you perform the preceding operations on a cluster with a large number of data, the entire cluster might need some time to completely replicate data to read-only nodes. During this period, the read-only nodes might not be able to provide services.
+> - Because of the special implementation of backup, the learner number of each label cannot exceed 1. Otherwise, it will generate duplicate data during backup.
+
+### 3. Use Follower Read to read data from read-only nodes
+
+#### 3.1 Use Follower Read in TiDB
+
+To read data from read-only nodes when using TiDB, you can set the system variable [`tidb_replica_read`](/system-variables.md#tidb_replica_read-new-in-v40) to `learner`:
+
+```sql
+set tidb_replica_read=learner;
+```
+
+#### 3.2 Use Follower Read in TiSpark
+
+To read data from read-only nodes when using TiSpark, you can set the configuration item `spark.tispark.replica_read` to `learner` in the Spark configuration file:
+
+```
+spark.tispark.replica_read learner
+```
+
+#### 3.3 Use Follower Read when backing up cluster data
+
+To read data from read-only nodes when backing up cluster data, you can specify the `--backup-replica-read-label` option in the br command line. Note that when running the following command in shell, you need to use single quotes to wrap the label to prevent `$` from being parsed.
```

Comment:
```suggestion
To read data from read-only nodes when backing up cluster data, you can specify the `--replica-read-label` option in the br command line. Note that when running the following command in shell, you need to use single quotes to wrap the label to prevent `$` from being parsed.
```

I just updated the Chinese version